README.md

```diff
@@ -9,3 +9,86 @@ $ bower install gaia-components/gaia-list
 ## Examples
 
 - [Example](http://gaia-components.github.io/gaia-list/)
+
+## Usage
```

Comment:
There's room for syntax improvement but I'd rather go through other patch to fix it. Just document what we got now.